### PR TITLE
Event source started before events are processed (simple/naive syncing approach)

### PR DESCRIPTION
**Pull request by csviri**

### CODE REVIEW

File: operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/EventMarker.java [hunk 1]
```diff
@@ -72,10 +77,18 @@ public boolean eventPresent(ResourceID resourceID) {
...
     var actualState = getEventingState(resourceID);
-    return actualState == EventingState.NO_EVENT_PRESENT;
+    return actualState == NO_EVENT_PRESENT;
   }
 
   public void cleanup(ResourceID resourceID) {
     eventingState.remove(resourceID);
   }
+
+  public List<ResourceID> resourceIDSWithEventPresent() {
```
[the remark this answers is not in the document]
Comment (csviri) [reply operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/EventMarker.java]: changed.

**Review by metacosm — APPROVED** [review]

File: operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/SyncingEventSource.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package io.javaoperatorsdk.operator.processing.event.source;
+
+public interface SyncingEventSource {
```
Comment (metacosm) [inline operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/SyncingEventSource.java:3]: Actually, I don't see where that's being used?

**Review by metacosm — CHANGES_REQUESTED** [review]

File: operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/SyncingEventSource.java (hunk 2 above)
Comment (csviri) [reply operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/SyncingEventSource.java:3]: hmm I though I deleted it, 1 sec

Comment (csviri) [reply operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/SyncingEventSource.java]: deleted.

**Review by metacosm — APPROVED** [review]

**Pull request merged**